app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    args = command.split()
+    if len(args) < 3:
+        raise FileNotFoundError("No file name specified.")
+
+    file1 = command.split()[1]
+    file2 = command.split()[2]
```

Comment:
Don't use split a few times. You can use variable args here nad unpucking
```suggestion
    file1, file2 = args[1:]
```